examples/todo/celest/functions/greeting.dart

```diff
@@ -0,0 +1,10 @@
+// Cloud functions are top-level Dart functions defined in the `functions/`
```

Comment:
Since you are not using this API file, please delete it as well.

Reply:
Done